docs/create_new_dev_env.md

```diff
@@ -81,4 +81,27 @@ image for all of them, by pressing :material-keyboard-return:.
 - deployer:`axemsolutions/stlink_org:latest`
 - test framework:`dummy_test:latest`
 
-That's it! Now you have a new Development Environment.
\ No newline at end of file
+That's it! Now you have a new Development Environment.
+
+## Share your new Development Environment
+
+A very important feature of DEM is to provide a reliable way to share Development Environments. To 
+make every member of an organization use the same toolset, the best way is to provide the 
+Development Environment from a catalog.  
+If you would like to share your Development Environment with someone directly, DEM has your back! 
+A Development Environment descriptor can be exported to a JSON file, which then can be loaded to 
+another host. 
+
+!!! note
+
+    The exported JSON file only contains the Development Environments descriptor. The other host 
+    needs access to all required registries for successful installation.
+
+If you'd like to share the newly created alt_env, first export it:
+
+    dem export alt_env
+
+This will create the alt_env.JSON file in your current directory.  
```

Comment:
export won't create the file with the extension, only it was explicitly given as a parameter